src/components/backend-ai-credential-view.ts

```diff
@@ -995,6 +996,16 @@ export default class BackendAICredentialView extends BackendAIPage {
     return Math.round(gigabyte * value);
   }
 
+  async _runAction() {
+    const action = location.search.split('action=')[1];
```

Comment:
@snaag This line will generate error when ?action GET parameter is missing.

<img width="269" alt="image" src="https://user-images.githubusercontent.com/949915/198499930-57e30f98-6071-4f43-80b1-049d4ddc389b.png">

Reply:
Oh right. Modified by adding code for checking.